ci-automation/ci_automation_common.sh

```diff
@@ -475,3 +475,34 @@ function list_files() {
     done
 }
 # --
+
+# Looks for ../scripts.patch, ../overlay.patch, ../portage.patch and
+# applies them to the current repo or the respective sub-module checkout.
+function apply_local_patches() {
+  local patch_files=(../scripts.patch ../overlay.patch ../portage.patch)
+  local patch_file
+  local patch_id
+  echo "Looking for local patches ${patch_files[*]}"
+  for patch_file in "${patch_files[@]}"; do
+    if [ "${patch_file}" = "../scripts.patch" ]; then
+      dirarg=()
```

Comment:
`dirarg` needs to be declared as local somewhere earlier.

Reply:
Thanks